docs/UserGuide.md

```diff
@@ -312,16 +325,20 @@ _Did you just receive a new customer? You can add the customer's details into Ho
 
 > Finds customers whose names contain _ANY_ of the given keywords.
 
-_Want to find details of a customer, but you do not remember the customer's full name?
-You can search for the customer using keywords by calling this command._
+_If you want to find details of a customer, but you do not remember the customer's full name,
+you can search for the customer using keywords by calling this command._
 
-**Format:** `customer find KEYWORD [MORE_KEYWORDS]`
+**Format:** `customer find KEYWORD [MORE_KEYWORDS...]`{.swift}
 
-<box background-color="#dff0d8" border-color="#d6e9c6" type="info" header="Note">
-    You must provide at least one keyword to search for the customer.
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+* `[]`{.swift} around a parameter indicates that it is optional.
+* You must provide at least one keyword to search for the customer. Multiple keywords can be provided.
 </box>
 
-**Example:** `customer find Gabriel Spencer`
+**Example:** `customer find Gabriel Spencer`{.swift} </br>
+Finds customers whose names contain _ANY_ of the given keywords `Gabriel`{.swift} and `Spencer`{.swift}.
```

Comment:
Would it be more apt to say their names have words that exactly match any keywords, e.g. Gabriel contains Gab but its not a match